routes/DonateRoute.js

```diff
@@ -0,0 +1,10 @@
+const router = require("express").Router();
+
+const { donateNow, flw_Webhook } = require("../controller/DonateController");
+
+router.post("/", donateNow);
+router.post("/flw-webhook", flw_Webhook);
```

Comment:
I assume only Flutterwave should be able to hit this endpoint. Do we want to move the validation into a middleware instead?

Reply:
Well, what do you feel works best? It's still an action that relates to Flutterwave.